Fix/docs payment interface (#159)
mollie/commercetools: docs/CreateOrder.md

## Overview

To create an order on Mollie, we get required parameters from the commercetools Cart and Payment. Payment must be added to a cart on commercetools before adding the initial transaction. Additionally, some parameters can be passed on the Payment's custom field `createPayment`. 

> Triggering create order on Mollie from commercetools should therefore happen in these consecutive steps:

Reviewer: ```suggestion
These steps must be followed to trigger order creation on mollie from commercetools:
```
